pages/translate.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import { TextInput } from '../components/TextInput';
+// you could move the fetching functionality to separate custom hook
+/// import { useTranslation } from '../utils/useTranslation';
+
+export default function Translate() {
+  const [data, setData] = useState(null)
+  const [isLoading, setLoading] = useState(false)
+  
+  const text = 'Hello';
```

Comment:
I moved your variables over here and used it then in API link, have a look how it's done. This might be useful when you think about the fetch function and what arguments it would need to have when you move it to `utils`